### ISSUE TITLE
6.1.2 introduces breaking bug for tailwind projects

### ISSUE THREAD

**Issue by mattjennings** [Siilwyn/css-declaration-sorter#191]
My project uses postcss + tailwind, and once 6.1.2 went live I noticed my builds were failing with this error:

```
[vite:css] Cannot read property 'includes' of undefined
file: /Users/mattjennings/dev/sveltekit-blog-template/src/app.css
> Cannot read property 'includes' of undefined
TypeError: Cannot read property 'includes' of undefined
    at /Users/mattjennings/dev/sveltekit-blog-template/src/app.css:1:1
    at /Users/mattjennings/dev/sveltekit-blog-template/node_modules/.pnpm/css-declaration-sorter@6.1.2_postcss@8.3.5/node_modules/css-declaration-sorter/src/main.cjs:333:46
    at /Users/mattjennings/dev/sveltekit-blog-template/node_modules/.pnpm/postcss@8.3.6/node_modules/postcss/lib/container.js:55:18
    at Root.each (/Users/mattjennings/dev/sveltekit-blog-template/node_modules/.pnpm/postcss@8.3.6/node_modules/postcss/lib/container.js:41:16)
    at Root.walk (/Users/mattjennings/dev/sveltekit-blog-template/node_modules/.pnpm/postcss@8.3.6/node_modules/postcss/lib/container.js:52:17)
    at processCss (/Users/mattjennings/dev/sveltekit-blog-template/node_modules/.pnpm/css-declaration-sorter@6.1.2_postcss@8.3.5/node_modules/css-declaration-sorter/src/main.cjs:327:7)
    at /Users/mattjennings/dev/sveltekit-blog-template/node_modules/.pnpm/css-declaration-sorter@6.1.2_postcss@8.3.5/node_modules/css-declaration-sorter/src/main.cjs:314:33
    at async LazyResult.runAsync (/Users/mattjennings/dev/sveltekit-blog-template/node_modules/.pnpm/postcss@8.3.6/node_modules/postcss/lib/lazy-result.js:431:15)
    at async compileCSS (/Users/mattjennings/dev/sveltekit-blog-template/node_modules/.pnpm/vite@2.5.3/node_modules/vite/dist/node/chunks/dep-1be34a63.js:34139:27)
    at async Object.transform (/Users/mattjennings/dev/sveltekit-blog-template/node_modules/.pnpm/vite@2.5.3/node_modules/vite/dist/node/chunks/dep-1be34a63.js:33729:50)
    at async ModuleLoader.addModuleSource (/Users/mattjennings/dev/sveltekit-blog-template/node_modules/.pnpm/rollup@2.56.2/node_modules/rollup/dist/shared/rollup.js:19708:30)
```

I believe this is the breaking change as it removed the safety check for `node.raws.before`:

https://github.com/Siilwyn/css-declaration-sorter/commit/528be4a47f99d3bae1bc4af88e0d60acffe6a251

The error occurs when `@tailwind base;` is used in the .css file. Removing it, or downgrading to 6.1.1 fixes the error. I also added a console log to that node before the error happens and this is the problematic one:

```
Comment {
  raws: {},
  type: 'comment',
  text: '! tailwindcss v2.2.9 | MIT License | https://tailwindcss.com',
  source: {
    start: { offset: 0, line: 1, column: 1 },
    input: Input {
      css: '@tailwind base;\n',
      hasBOM: false,
      file: '/Users/mattjennings/dev/sveltekit-blog-template/src/app.css',
      [Symbol(fromOffsetCache)]: [Array]
    },
    end: { offset: 14, line: 1, column: 15 }
  },
  parent: Root {
    raws: { semicolon: true, after: '' },
    type: 'root',
    nodes: [
      [Circular *1], [Comment], [Rule], [Rule],
      [Rule],        [Rule],    [Rule], [Rule],
      [Rule],        [Rule],    [Rule], [Rule],
      [Rule],        [Rule],    [Rule], [Rule],
      [Rule],        [Rule],    [Rule], [Rule],
      [Rule],        [Rule],    [Rule], [Rule],
      [Rule],        [Rule],    [Rule], [Rule],
      [Rule],        [Rule],    [Rule], [Rule],
      [Rule],        [Rule],    [Rule], [Rule],
      [Rule],        [Rule],    [Rule], [Rule],
      [Rule],        [Rule],    [Rule], [Rule],
      [Rule],        [Rule],    [Rule], [Rule],
      [Rule],        [Rule],    [Rule], [Rule],
      [Rule],        [Rule],    [Rule], [Rule],
      [Rule],        [Rule],    [Rule], [Rule],
      [Rule],        [Rule],    [Rule], [Rule],
      [Rule],        [Rule],    [Rule], [Rule],
      [Rule],        [Rule],    [Rule], [AtRule],
      [AtRule]
    ],
    source: { input: [Input], start: [Object] },
    lastEach: 64,
    indexes: { '64': 0 },
    _autoprefixerDisabled: false,
    _autoprefixerPrefix: false,
    _autoprefixerGridStatus: false,
    rawCache: {
      beforeDecl: '\n',
      indent: '\t',
      commentLeft: '',
      commentRight: ' ',
      beforeOpen: '',
      semicolon: true,
      colon: ':',
      beforeClose: '\n',
      beforeRule: '\n\n'
    },
    [Symbol(isClean)]: false,
    [Symbol(my)]: true
  },
  [Symbol(isClean)]: false,
  [Symbol(my)]: true,
  [Symbol(my)]: true,
  [Symbol(isClean)]: true
}
```

Hopefully that's enough info, otherwise I can create a repro for you if needed.

**Comment by xellDart** [comment]
I get same error

```
[vite:css] Cannot read property 'includes' of undefined
file: /Users/miguel/Documents/nebuia/widget_web/src/app.postcss
> Cannot read property 'includes' of undefined
TypeError: Cannot read property 'includes' of undefined
    at /Users/miguel/Documents/nebuia/widget_web/src/app.postcss:1:1
    at /Users/miguel/Documents/nebuia/widget_web/node_modules/css-declaration-sorter/src/main.cjs:333:46
```

**Comment by Siilwyn** [comment]
Thanks for the report! Should be fixed in 6.1.3.

**Comment by mattjennings** [comment]
Awesome, thanks for the quick fix!
